### PR TITLE
change page number

### PR DESCRIPTION
**Pull request by shangshanling0**
change page number to 4

### CODE REVIEW

**Comment by CapA0** [comment]
yeah, you are right, thanks.

File: config.py [hunk 1]
```diff
@@ -14,4 +14,4 @@ class Config(object):
     MAIL_PASSWORD = os.environ.get('MAIL_PASSWORD')
     ADMINS = ['your-email@example.com']
 
-    POSTS_PER_PAGE = 3
\ No newline at end of file
+    POSTS_PER_PAGE = 4
```
Comment (CapA0) [inline config.py]: this may be 5.

Comment (shangshanling0) [reply config.py]: ok

Comment (shangshanling0) [inline config.py]:
```suggestion
    POSTS_PER_PAGE = 5
```

**Pull request merged**